src/back/spv/instructions.rs

```diff
@@ -727,6 +727,22 @@ pub(super) fn instruction_switch(
     instruction
 }
 
+pub(super) fn instruction_select(
```

Comment:
this needs to be rebased now, we made all these to be constructor methods

Reply:
I have performed the rebase.